packages/app/src/cli/models/app/loader.ts

```diff
@@ -219,6 +220,7 @@ class AppLoader {
       specifications: this.specifications,
     })
     const {directory, configuration, configurationLoadResultMetadata, configSchema} = await configurationLoader.loaded()
+    this.fullAppConfiguration = configuration
```

Comment:
This is the first time (I think) we need data from other modules in order to create another. I decided to set the full config to a property to limit the argument drilling. Is there a better way of doing this?

Reply:
The codebase is a bit mess... I can't think of a better approach without a big refactor.

The problem is that I'll need to add another parameter to the transform functions (the beta flags) in https://github.com/Shopify/cli/pull/3392 🙈